ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/rlpx/RlpxAgent.java

```diff
@@ -329,6 +335,16 @@ private void handleIncomingConnection(final PeerConnection peerConnection) {
       return;
     }
 
+    // Disconnect if the fraction of wire connections initiated by peers is too high to protect
+    // against eclipse attacks
+    if (limitRemoteWireConnectionsEnabled && !doTheFractionOfRemoteConnectionsAllowsNewOne()) {
```

Comment:
for readability I would change the function to return a `true` if the ratio is too high, so we don't have a negation '!' in the boolean logic.  We also negate it inside the method.  Also, the name would need to change.

```suggestion
    if (limitRemoteWireConnectionsEnabled && remoteConnectionExceedsLimit()) {
```

Reply:
Ok.